test/parallel/test-util-emit-experimental-warning.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+
+// This test ensures that the emitExperimentalWarning in internal/util emits a
+// warning when passed a unsupported feature and that it simply returns when
+// passed a supported feature.
+
+const { emitExperimentalWarning } = require('internal/util');
+
+process.on('warning', common.mustCall((warning) => {
+  assert(/is an experimental feature/.test(warning.message));
+}, 2));
+emitExperimentalWarning('feature1');
+emitExperimentalWarning('feature1');
```

Comment:
Could you add, at the end of the line, `// should not warn`

Reply:
@apapirovski : Thanks for the feedback. I've updated the PR. Kindly review now.